app/views/otherbusinessinvolvements/OtherBusinessName.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.data.Form
+@import play.api.mvc.Request
+@import play.api.i18n.Messages
+@import config.FrontendAppConfig
+@import views.html.layouts
+
+@this(
+        layout: layouts.layout,
+        errorSummary: components.errorSummary,
+        form: FormWithCSRF,
+        inputText: components.inputText,
+        button: components.button
+)
+
+@(businessNameForm: Form[String], index: Int)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@layout(pageTitle = Some(title(businessNameForm, messages("obi.otherBusinessName.heading"))), backLink = true) {
+
+    @errorSummary(businessNameForm.errors)
+
+    @form(action = controllers.registration.otherbusinessinvolvements.routes.OtherBusinessNameController.submit(index)) {
+        @inputText(
+            businessNameForm,
+            id = "otherBusinessName",
+            name = "otherBusinessName",
+            label = messages("obi.otherBusinessName.heading"),
+            isPageHeading = true
+        )
+
```

Comment:
Can you add 'class = govuk-input govuk-input--width-20' to limit the input textbox size?